app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -707,6 +710,8 @@ class BrowserTabFragment :
             contentScopeScripts.onResponse(it)
         }
 
+    private val jsOrientationHandler = JsOrientationHandler()
```

Comment:
Please use DI for this.

Reply:
Yep yep - please see https://github.com/duckduckgo/Android/pull/4027#discussion_r1433339787